[2024-05-18T15:34:16.618+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.seceond_task manual__2024-05-18T15:28:03.593822+00:00 [queued]>
[2024-05-18T15:34:16.629+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.seceond_task manual__2024-05-18T15:28:03.593822+00:00 [queued]>
[2024-05-18T15:34:16.630+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-05-18T15:34:16.648+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): seceond_task> on 2024-05-18 15:28:03.593822+00:00
[2024-05-18T15:34:16.658+0000] {standard_task_runner.py:60} INFO - Started process 2564 to run task
[2024-05-18T15:34:16.662+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'seceond_task', 'manual__2024-05-18T15:28:03.593822+00:00', '--job-id', '367', '--raw', '--subdir', 'DAGS_FOLDER/frist-dag.py', '--cfg-path', '/tmp/tmpkuagqufw']
[2024-05-18T15:34:16.669+0000] {standard_task_runner.py:88} INFO - Job 367: Subtask seceond_task
[2024-05-18T15:34:16.733+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v2.seceond_task manual__2024-05-18T15:28:03.593822+00:00 [running]> on host 7b23a6949395
[2024-05-18T15:34:16.835+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='seceond_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-18T15:28:03.593822+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-18T15:28:03.593822+00:00'
[2024-05-18T15:34:16.837+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-18T15:34:16.839+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'pwd']
[2024-05-18T15:34:16.858+0000] {subprocess.py:86} INFO - Output:
[2024-05-18T15:34:16.861+0000] {subprocess.py:93} INFO - /tmp/***tmpej68ejct
[2024-05-18T15:34:16.862+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-18T15:34:16.896+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=seceond_task, execution_date=20240518T152803, start_date=20240518T153416, end_date=20240518T153416
[2024-05-18T15:34:16.954+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-05-18T15:34:16.976+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
